zkvm/vm/evm/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 revm = { git = "https://github.com/powdr-labs/revm", branch = "serde-no-std", default-features = false, features = [ "serde" ] }
-powdr-riscv-runtime = { git = "https://github.com/eigmax/powdr", branch = "main" }
```

Comment:
I use a fork here because I want to dump proof and its Circom(stark verifier), so I can do recursive proof then. Do you support dumping those two files now?

Reply:
which files do you need dumped? You mean for `powdr` or the runtime crate?

Reply:
here https://github.com/powdr-labs/powdr/compare/main...eigmax:powdr:main#diff-1dc9d1b77e4f7d829044d5631aca37a8e23774e1d451edefff357662ec311d56R162

Reply:
The Circom file(as the second proof in my patch) is necessary to do recursive proof, but I have no idea how to place it properly, and we haven't finished the proof network, so that's why I still use the patch.

Reply:
ah so it's not the runtime crate, but the actual powdr repo, right?

Reply:
yes. I think powdr needs to produce 2 proofs(starkproof and its circom) when eStark used?

Reply:
Yea let's sync on that in DMs, see how we can upstream that